src/lib/y2storage/blk_device.rb

```diff
@@ -218,19 +218,19 @@ def udev_full_ids
     alias_method :create_filesystem, :create_blk_filesystem
 
     # @!method create_bcache(name)
-    #   Creates backing device build on current block device with given name.
+    #   Creates Bcache device with given name over the current block device.
```

Comment:
maybe this is more clear saying "Creates a bcache device with the given name, using this device as backing device"